### ISSUE TITLE
Feature request: version numbers

### ISSUE THREAD

**Issue by Bellfalasch** [enonic/module-skeleton#2]
Something that we really need in the old STK was version numbering, so we could easier know if we needed to update the STK or not when visiting different clients. Since each file is updated separately this becomes a problem fast. Could the Skeleton use numbering this way? Same way as we number EXP. X.0.0 is API breaking, 0.0.X is bugfix/minor, etc.

**Comment by Michael-Lazell** [comment]
Agreed, that would be very helpful. 0.0.X is bug fix, 0.X is new features, and X is breaking change. At this stage, the framework has a lot of evolving to do. Also, once you start a project with it, how do you update it? Before you could overwrite the STK folder, but now there is more to it.

**Comment by Bellfalasch** [comment]
Indeed, I went through each and every file manually and replaced my local code. And ofc with files I've already modified (my parts and templates) extra care was needed.
